mender-update/standalone/standalone.cpp

```diff
@@ -272,6 +273,14 @@ ResultAndError Commit(context::MenderContext &main_context) {
 
 	update_module::UpdateModule update_module(main_context, data.payload_types[0]);
 
+	if (data.payload_types[0] == "rootfs-image") {
```

Comment:
IMHO, these payload type strings should be constants somewhere, not just magic string values scattered in multiple places in the code.

Reply:
Some of them can, but not all

Reply:
They can be user defined

Reply:
Sure, but the string literals in the code can all be constants defined somewhere, right?

Reply:
Like I said, some of them can :)

Reply:
Well, the ones that are string literals are not user defined and can be constants.

Reply:
Hmm, yes, I see your point. There are a lot of these in the code though, and they are not always easily replaced by a constant because they are combined with other strings:

```
mender-update/cli/cli_test.cpp:	EXPECT_TRUE(VerifyProvides(tmpdir.Path(), R"(rootfs-image.version=test
mender-update/cli/cli_test.cpp:rootfs-image.checksum=f2ca1bb6c7e907d06dafe4687e579fce76b37e4e93b7605022da52e6ccc26fd2
mender-update/cli/cli_test.cpp:	string update_module = path::Join(tmpdir.Path(), "rootfs-image");
...
mender-update/update_module/v3/update_module_test.cpp:	      "type": "rootfs-image"
mender-update/update_module/v3/update_module_test.cpp:	    "rootfs-image.checksum":
mender-update/update_module/v3/update_module_test.cpp:	    "rootfs-image.version": "test-artifact"
mender-update/update_module/v3/update_module_test.cpp:	    "rootfs-image.*"
```

This is just a handful of examples, there are many more.

I don't think they should all be replaced with a constant, because it would actually make it harder to see what the code is about.

The `rootfs-image` type is essentially an API string at this point. It is not going to change. So I think we might as well leave it alone for readability.